src/WCSimPrimaryGeneratorMessenger.cc

```diff
@@ -130,6 +175,10 @@ WCSimPrimaryGeneratorMessenger::~WCSimPrimaryGeneratorMessenger()
   delete injectorTimeCmd;
   delete openingAngleCmd;
   delete injectorWavelengthCmd;
+  delete lightInjectorCmd;
+  delete lightInjectorIdxCmd;
+  delete lightInjectorNPhotonsCmd;
+  delete lightInjectorFilenameCmd;
```

Comment:
```suggestion
  delete lightInjectorFilenameCmd;
  delete lightInjectorModeCmd;
```
Think you're missing one